terminus/generators/monolane_id_mapper.py

```diff
@@ -0,0 +1,120 @@
+from collections import OrderedDict
+from math import degrees
+
+from city_visitor import CityVisitor
+from models.road_intersection_node import RoadIntersectionNode
+
+
+class MonolaneIdMapper(CityVisitor):
+    """Simple city visitor that generates the Monolane point names ("id's").
+
+    This class refers to "road nodes", which in this case is a unique
+    combination of a road (a street or a trunk) with a node.
+    The node might be a RoadIntersectionNode, in which case the node might
+    be associated with more than one road.
+    For the purposes of Monolane, this is not OK, so we create a point name per
+    road-node pair.
+    """
+
+    def run(self):
+        self.point_name_by_road_node_dict = {}
+        self.intersection_index = 0
+        self.intersection_index_by_node = {}
+        self.monolane_points = OrderedDict()
+        super(MonolaneIdMapper, self).run()
+
+    def road_node_id(self, road, node):
+        """Return a hashable unique identifier for a road node."""
+        if road.name is None:
+            print(road)
```

Comment:
Not sure if this is supposed to happen or not. Can you replace this for either an exception if it should break or a `logger.debug` if it should remain for debugging purposes but not fail?